README.md

```diff
@@ -4,6 +4,24 @@
 [gitter-image]: https://badges.gitter.im/open-telemetry/opentelemetry-python.svg
 [gitter-url]: https://gitter.im/open-telemetry/opentelemetry-python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
+We hold regular meetings. See details at [community page](https://github.com/open-telemetry/community#python-sdk).
```

Comment:
Seems like we could move this out of the community repo too.

Reply:
mmm, shouldn't that be "python SIG" instead of "python SDK"?

Reply:
It should, but "SDK" had a different meaning when that doc was first written. We can fix that here by just moving the content into this doc.

Reply:
I go back and forth. It's a little nice to see all the meetings in one list. It's not always evident what meetings do or do not occur.

Reply:
I agree with @toumorokoshi. I like seeing a list of all meetings and details in one place. With that being said, we can change python-sdk to python-sig in the community repo.

Reply:
We could put it in both places. It seems a bit ridiculous to direct people to the community repo when we could just say "every thursday at 8AM PT" here.

https://github.com/open-telemetry/community/issues/233 also says we ought to move the meeting schedule here:

> Similarly, the SIG information (schedule of meetings, etc.) can be also moved from the README in this repo to the READMEs in the respective repos. This repo will only have the links to information.